include/oneapi/dpl/pstl/numeric_impl.h

```diff
@@ -206,13 +206,16 @@ __brick_transform_scan(_RandomAccessIterator __first, _RandomAccessIterator __la
                        /*is_vector=*/::std::true_type) noexcept
 {
 #if (_PSTL_UDS_PRESENT || _ONEDPL_UDS_PRESENT)
-    return __unseq_backend::__simd_scan(__first, __last - __first, __result, __unary_op, __init, __binary_op,
-                                        _Inclusive());
-#else
+    if (::std::is_same_v<_Inclusive, ::std::true_type> ||
```

Comment:
From my point of view better to write something like ```_Inclusive{}```

Reply:
Done

Reply:
We deliberately throw out the incoming argument with type `_Inclusive`, then here we would create a new one using the type.  Should we not just name the incoming argument `__is_inclusive` and use it?

Reply:
I suppose we probably should also replace the `_Inclusive()` in the call itself with such a variable.